app/main/lib/image_hash.py

```diff
@@ -20,14 +26,41 @@ def compute_ahash(im):
     :returns: Imagehash.ImageHash
   """
   return imagehash.average_hash(ensure_pil(im))
-
 def compute_phash(im):
   """Compute perceptual hash using ImageHash library
     :param im: Numpy.ndarray
     :returns: Imagehash.ImageHash
   """
   return imagehash.phash(ensure_pil(im))
 
+def compute_pdq(im):
+  """Compute perceptual hash using ImageHash library
+    :param im: Numpy.ndarray
+    :returns: Imagehash.ImageHash
+  """
+  with io.BytesIO() as output:
+    ensure_pil(im).save(output, format="GIF")
+    pdq_hasher = PDQHasher()
+    hash_and_qual = pdq_hasher.fromBufferedImage(output)
+    print("hash_and_qual")
```

Comment:
Let's comment all of these `print` statements. Anything really important for logging should get logged with the `Logger`

Reply:
I've commented these now.